### ISSUE TITLE
Panic evaluating expression

### ISSUE THREAD

**Issue by jdm** [jdm/libmzx#65]
https://www.digitalmzx.com/show.php?id=2420
```
     Running `target/debug/capture image.png output.json unzipped/Thud/AA_BKZX_THUD.mzx`
Capturing board 2: BOARD 1
thread 'main' panicked at 'attempt to divide by zero', /Users/jdm/src/libmzx/src/expression.rs:371:56
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by jdm** [comment]
https://github.com/jdm/libmzx/blob/f860ecbfd36d4e986f144152f311010ebbad60b8/src/expression.rs#L371

**Comment by jdm** [comment]
This comes from `('Xmo'/(&abs('xmo')&))` but I should check what megazeux does in this case.

**Comment by jdm** [comment]
This happens on almost every board in https://www.digitalmzx.com/show.php?id=2373, and on the win board in particular it's easy to see that the expression evaluates to 0 in the end.

**Comment by jdm** [comment]
Fixed in c3252f0f95f205eff7fe2da95072653155ecaa55.
